Gems/Atom/RPI/Assets/ShaderLib/Atom/RPI/ShaderResourceGroups/DefaultDrawSrg.azsli

```diff
@@ -28,6 +28,8 @@ ShaderResourceGroup DrawSrg : SRG_PerDraw
 {
     // This SRG is unique per draw packet
     uint m_uvStreamTangentBitmask;
+    int m_materialTypeId;
```

Comment:
Does this make more sense to be part of ObjectSRG?

Reply:
good point. Moved to the ObjectSrg